prerelease clarify (#40)
noxonsu/tools.onout.org: contributing.md

# шаг 4 пререлиз
1. добавить в dash.onout.org в раздел presale . https://github.com/noxonsu/dash.onout.org/blob/main/src/constants/index.ts#L130 (скопировать данные, самое главное id чтоб был различный от других продуктов)
2. разместить на поддомене версию с тестнет режимом, добавить в dash.onout.org ссылку на demo. Позже тестнет нужно заменить на мейннет при этом сделать, чтоб работало с токенами SWAP и USDT. т.е. наши токены в мейннете должны использоваться для демонстрации как работает продукт. 
3. записать два видео. Как пользоватся продуктом со стороны юзера (какой бенифит у юзера? не нужно растягивать как работает, сейчас цель что то опубликовать, потом уже по обратной связи доработать) и как пользоватся со стороны админа (какой бенифит у админа, как ему зарабатывать, куда воткнуть комсу?). Если админка еще не готова, то делать только то как пользуется юзер. 

> 4. отправить ссылку на демо и ссылку на видео в дискорд, "Hello, we've published alfa version. You can seee demo here: https:// ... . If you have any questions please use this chat. Presale is also available at https://dash.onout.org/#/presale . Current price set to 1000$. Next step is to develop admin panel"

Reviewer: ```suggestion
4. отправить ссылку на демо и ссылку на видео в дискорд, "Hello, we published an alfa version. You can watch demo here: https:// ... . If you have any questions please use this chat. Presale is also available at https://dash.onout.org/#/presale . Current price - $1000. Next step is an admin panel development"
```
